packages/edit-site/src/components/page-pages/index.js

```diff
@@ -273,6 +273,16 @@ export default function PagePages() {
 		[ totalItems, totalPages ]
 	);
 
+	const { frontPageId, postsPageId } = useSelect( ( select ) => {
```

Comment:
Potential follow-up: I think this one can potentially move to the "render" function instead (or should I say render component) (we should treat them as component in DataViews first though) which will make sure it only runs if the field is actually rendered.

Reply:
Title is always rendered in pages, so it would be the same..

Reply:
Yes, but still :) conceptually it makes sense for this to be within the field.